### ISSUE TITLE
Query log: Offer ability to determine number of rows shown.

### ISSUE THREAD

**Issue by twcau** [AdguardTeam/AdGuardHome#1849]
### Problem Description
At present, Query log shows paginated data from querylog.json.

However, the limit of rows shown is hard coded. The user cannot choose how many rows to show.

This is inconsistent with other parts of the Management Interface, for example Settings > Client Settings. 

### Proposed Solution
To both the top and bottom of Query Log, add a drop-down for the user to choose the number of rows shown on the screen.

### Alternatives Considered
No plausible alternatives identified.

### Additional Information
N/A

**Comment by mHatsune** [comment]
Duplicate of #1851

**Comment by twcau** [comment]
> Duplicate of #1851

Wrong, not a duplicate at all. They are two entirely different requests, and the only relationship is that they are requests for the same part of the app.

Strongly recommend that you re-read both before being so quick to comment that something is a dupe.

**Comment by mHatsune** [comment]
Well, i noticed that and same proposed solution mentioned on #1851 however removed 'duplicated' mark.
Sorry if that made you disappoint

**Comment by Sil3ntVip3r** [comment]
Revalidated against the current UI: the original rows-per-page request has
been superseded by infinite scrolling.  `LogTable` explicitly disables
pagination, fetches fixed batches of 20, and appends each batch to the
displayed results.

There is therefore no current “page” whose visible row count can be
configured.  I recommend closing this as superseded by the broader
pagination/storage work in #3426 and #2290.

If the desired behavior is instead a configurable infinite-scroll fetch size,
the issue should be restated that way because it is a performance preference
rather than a visible-row selector.
